buildSrc/src/main/kotlin/android-library-convention.gradle.kts

```diff
@@ -1,32 +1,16 @@
-@file:Suppress("UnstableApiUsage")
-
 plugins {
     id("com.android.library")
     id("kotlin-android")
     id("maven-publish-android")
 }
 
 android {
-    compileSdk = (Versions.androidCompileSdk)
+    compileSdk = Versions.androidCompileSdk
+    buildToolsVersion = Versions.androidBuildToolsVersion
 
     defaultConfig {
         minSdk = Versions.androidMinSdk
-    }
-
-    compileOptions {
-        sourceCompatibility = JavaVersion.VERSION_1_8
-        targetCompatibility = JavaVersion.VERSION_1_8
-    }
-    kotlinOptions {
-        jvmTarget = "1.8"
-
-        // avoid kotlin_module clash for projects with the same name, e.g. 'ui' & 'routes:ui"
-        // https://discuss.kotlinlang.org/t/dealing-with-kotlin-module-conflict-when-building-apk-for-project-with-non-unique-project-names/11247
```

Comment:
The motivation for these build arguments hasn't changed, why do you suggest removing them?